target/linux/ramips/dts/mt7620a_netgear_ex6120.dts

```diff
@@ -0,0 +1,58 @@
+/* This file is released into the public domain */
+
+/dts-v1/;
+
+#include "mt7620a_netgear_ex3700_ex6130.dtsi"
+
+/ {
+	compatible = "netgear,ex6120", "ralink,mt7620a-soc";
+	model = "Netgear EX6120";
+
+	aliases {
+		led-boot = &led_power_green;
+		led-failsafe = &led_power_green;
+		led-running = &led_power_green;
+		led-upgrade = &led_power_green;
```

Comment:
Just found that there already was a (quite unsatisfying) discussion on this device. In that PR, somebody had a look at the manual and found that the LED assignment should different to match vendor behavior:
https://github.com/openwrt/openwrt/pull/2870#discussion_r399712110

So, led-boot should be changed to &led_power_red, and one should consider whether led-failsafe and led-upgrade should be changed as well.

Reply:
Sorry to bring the news but this is, for a reason, not the first attempt of someone having a physical EX6120 unit and the time and base knowledge to make this into a commit for the greater good of overall hardware support of openwrt, only to be chased away for minor BS reasons that could easily be fixed afterwards/concurrently by an experienced maintainer in a tiny fraction of the time.  I've now spent several hours plus 6 hours of compile time for a device that I do even not intend to operate, but happens to be in my possession. This is far beyond my time budget for this, I intend to finish the process asap for the pure experience of it and possibly never contribute to openwrt ever again.

The device boots with an amber LED on stock firmware and until the point of testing the triggers for LEDs as described above I have never even seen the red LED, despite repairing almost a dozen of these in various damaged states from hardware defects to failed fw upgrades to god knows what people do with innocent AP hardware/software. Red is no boot color and solid amber for extended periods of time is an indicator of a boot loop that requires TFTP flashing. I'll add a colorful mix for those states that might not work if that makes you happy.

Reply:
> be fixed afterwards/concurrently by an experienced maintainer in a tiny fraction of the time

No. There will be nobody cleaning up behind us, and that's why I try to make things right in the first. We have enough old devices where people just copied together code and nobody ever really checked things.

Despite, I have given you quite precise pointers on what should be changed, so implementation could be done with limited or no research.

> for a device that I do even not intend to operate, but happens to be in my possession.

I do not even have the device, and still invest my time to review your PR and try to make it ready for merge.

> This is far beyond my time budget for this, I intend to finish the process asap for the pure experience of it and possibly never contribute to openwrt ever again.

That's sad. But providing device support takes its time and I haven't even asked for the ugly stuff (i.e. flashing back OEM firmware to check the MAC assignment). I'm sorry, but I won't accept an incomplete PR just to make you happy.

> The device boots with an amber LED on stock firmware and until the point of testing the triggers for LEDs as described above

So that sounds like you should use
led-boot = &led_power_amber
in aliases and
```
		led_power_amber: power_amber {
			label = "ex6120:amber:power";
			gpios = <&gpio2 28 GPIO_ACTIVE_LOW>;
		};
```
which should produce a blinking amber power led at boot and a solid green power led after boot. Or is the power_amber led not amber after all?


I'm sorry that you feel bugged, but as stated earlier not making things properly in the first place will create additional work for other people later, and we don't care now why should they care later?